core/primitives/base58_codec.cpp

```diff
@@ -96,7 +100,7 @@ namespace kagome::primitives {
     // Copy result into output vector.
     common::Buffer res{};
     res.reserve(zeroes + (b256.end() - it));
-    std::fill_n(res.begin(), zeroes, 0);
+    while (zeroes-- != 0) res.putUint8(0);
```

Comment:
Why? It does the same. Do you sure if it does more efficiently

Reply:
previous solution didn't work, I just took the original one

Reply:
this whole codec is just a modified copypaste from bitcoin